rllib/core/rl_module/tf/tf_rl_module.py

```diff
@@ -42,14 +42,28 @@ def get_state(self) -> Mapping[str, Any]:
     def set_state(self, state_dict: Mapping[str, Any]) -> None:
         self.set_weights(state_dict)
 
+    @override(RLModule)
+    def _weights_relative_path(self) -> pathlib.Path:
+        return pathlib.Path("module_state/module_state")
```

Comment:
why two module_state folders?

Reply:
so basically whats happening here is if you try to generate a tf checkpoint, you do so with the name of that checkpoint, say like "module_state". TF checkpoints don't have a file extension because they generate 3 or 4 files and this could make the directory structure get pretty cluttered / hard to interpret (at least for me). So instead I put the generated files inside of another directory, called "module_state"

I should add an extra docstring that way developers understand what's going on here.

Reply:
fixed

Reply:
ok this no longer applies exactly since the torch and tf weights will both be stored under a folder called "module_state'